siliconcompiler/apps/_common.py

```diff
@@ -0,0 +1,69 @@
+import glob
+import os
+
+
+def manifest_find_switches():
```

Comment:
Nit: maybe just `manifest_switches()` would be more accurate as their is no finding going on.